controllers/hosting/hostingdeployment_controller_test.go

```diff
@@ -318,6 +359,19 @@ var _ = Describe("Reconciling a HostingDeployment that exists", func() {
 					})
 				})
 
+				Context("EndpointName is defined", func() {
```

Comment:
nit: A better message might be something like "deterministic endpoint name" since that's what we have been calling it. 
I can imagine someone being a little confused by this context in a few months.